src/trust_anchor_util.rs

```diff
@@ -52,3 +60,55 @@ pub fn generate_code_for_trust_anchors(name: &str,
 
     decl + &value
 }
+
+/// Parses a v1 certificate directly into a TrustAnchor.
+fn parse_cert_v1<'a>(cert_der: untrusted::Input<'a>)
+                     -> Result<TrustAnchor<'a>, Error> {
+    /* The top level is a:
+     *
+     * Certificate ::= SEQUENCE {
+     *   certificateInfo CertificateInfo,
+     *   signatureAlgorithm AlgorithmIdentifier,
+     *   signature BIT STRING }
+     *
+     * CertificateInfo ::= SEQUENCE {
+     *   version [0] Version DEFAULT v1988,
+     *   serialNumber CertificateSerialNumber,
+     *   signature AlgorithmIdentifier,
+     *   issuer Name,
+     *   validity Validity,
+     *   subject Name,
+     *   subjectPublicKeyInfo SubjectPublicKeyInfo }
+     */
```

Comment:
Rather than copy the documentation from the RFC, just cite this documentation with a link.

We copy/pasted the ASN.1 definitions into the source code for mozilla::pkix, but for webpki I decided that was a mistake; it's better to read them out of the source document if we want to read them at all. That way we don't have to worry about whether we copy/pasted the correct thing and/or copy/pasted wrongly. In this case, https://tools.ietf.org/html/rfc5280#section-4.1 is a good link.